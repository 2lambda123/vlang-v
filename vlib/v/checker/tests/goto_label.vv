fn f() {
	goto l
	l:
}

fn g() {
	l2:
	goto l
	goto l2
	goto l3
}

