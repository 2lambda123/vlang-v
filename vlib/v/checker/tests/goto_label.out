vlib/v/checker/tests/goto_label.vv:8:7: error: unknown label `l`
    6 | fn g() {
    7 |     l2:
    8 |     goto l
      |          ^
    9 |     goto l2
   10 |     goto l3
vlib/v/checker/tests/goto_label.vv:10:7: error: unknown label `l3`
    8 |     goto l
    9 |     goto l2
   10 |     goto l3
      |          ~~
   11 | }
   12 |
